iidm/iidm-mergingview/src/test/java/com/powsybl/iidm/mergingview/LineAdapterTest.java

```diff
@@ -104,19 +104,6 @@ void testSetterGetter() {
         assertEquals(lineRef.checkTemporaryLimits2(0.0f, LimitType.CURRENT), lineAdapted.checkTemporaryLimits2(0.0f, LimitType.CURRENT));
         assertEquals(lineRef.checkTemporaryLimits2(LimitType.CURRENT), lineAdapted.checkTemporaryLimits2(LimitType.CURRENT));
 
-        assertEquals(lineRef.checkPermanentLimit(Branch.Side.ONE, 0.0f), lineAdapted.checkPermanentLimit(Branch.Side.ONE, 0.0f));
```

Comment:
You cannot delete that, it should continue to work.

Reply:
These tests do not compile anymore, as I cleaned the corresponding deprecated methods.